src/Components/Navbar/navbar.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../Styles/index.scss';
```

Comment:
i dont think you should  import variables in every place. There definitely is a way how to define variables globally, import them globally and access in all scss files without having the import. Try google it/ ask chat gpt :-D

Reply:
add this code to vite.config.js css: {
    preprocessorOptions: {
      scss: {
        additionalData: `@import "@/styles/variables.scss";`,
      },
    },
  },